### ISSUE TITLE
location services can no longer be switched on

### ISSUE THREAD

**Issue by herrdeh** [microg/android_packages_apps_UnifiedNlp#35]
That quite likely is not a UnifiedNlp issue - but hopefully somebody here has an idea what to do...

After 2 weeks of happily using a gapps-free UnifiedNLP I suddenly can not any longer switch on location services at all. The slider just sweeps back - that's all.  I re-installed google play services - but that doesnt change anything.

What could I do? - I've got titanium backups of everything - what components should I restore?

**Comment by mar-v-in** [comment]
Did you install any relevant updates or software since it worked? maybe some energy saver tool? are you using UnifiedNLP with the Xposed plugin? Additional system details would be helpful

**Comment by herrdeh** [comment]
No, nothing at all. No new software, no updates, no energy saver. The last 
bit of software to be installed was Mozilla Stumbler, and location services 
worked nicely after the install.

I'm using slimkat 4.4.4 (9.0) on a Huawei y300; dont have a xposed plugin 
or any other xposed component installed.
